src/main/java/ca/uhn/fhir/jpa/starter/JpaRestfulServer.java

```diff
@@ -24,8 +28,15 @@ protected void initialize() throws ServletException {
     super.initialize();
 
     // Add your own customization here
+    
+    /* Custom ServerConformanceProvider will be triggered when fhir version is R4 and Oauth is enabled. */
+    if (FHIR_VERSION.equals(FhirVersionEnum.R4.name()) && Boolean.parseBoolean(OAUTH_ENABLED)) {
```

Comment:
Should this extension only appear in R4?

Any idea why the "http://hl7.org/fhir/StructureDefinition/capabilitystatement-websocket" extension appears in the metadata? You didn't add any code that would cause that, right?

Reply:
This extension should only appear for R4, as we have applied security to R4 server.
and I think this is the default extension "http://hl7.org/fhir/StructureDefinition/capabilitystatement-websocket" because it comes before we applied these changes

Reply:
Ok. It was there before you added the oauth-urls extension.